doc/qbk/quickref.xml

```diff
@@ -80,6 +80,9 @@
         <bridgehead renderas="sect3">Functions</bridgehead>
         <simplelist type="vert" columns="1">
           <member><link linkend="url.ref.boost__urls__operator_lt__lt_">operator&lt;&lt;</link></member>
+          <member><link linkend="url.ref.boost__urls__format">format</link></member>
+          <member><link linkend="url.ref.boost__urls__format_to">format_to</link></member>
+          <member><link linkend="url.ref.boost__urls__arg">arg</link></member>
```

Comment:
This is not in alphabetical order and why did you add `arg`?

Reply:
To explain what this public function does?

Reply:
yeah, alphabetical please though